lib/node_modules/@stdlib/math/base/special/lcmf/benchmark/benchmark.js

```diff
@@ -0,0 +1,108 @@
+/**
```

Comment:
Looks like a copy-paste error has been occurred in this file.

Reply:
You can rectify this, as this is unresolved.

Reply:
I rectified the same on my end
it's showing right on my side

Reply:
[https://github.com/Harsh-Mathur-1503/stdlib/blob/lcmf/lib/node_modules/@stdlib/math/base/special/lcmf/benchmark/benchmark.js](url)

Reply:
@gunjjoshi I think it should be resolved now ?